project/AppDependencies.scala

```diff
@@ -17,8 +17,8 @@ object AppDependencies {
     "org.scalatest"           %% "scalatest"                   % "3.0.8"   % "test",
```

Comment:
"bootstrap-backend-play-27" and "hmrc-mongo-test-play-27" for "test" needs to be updated to:
```suggestion
    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.3.0" % "test",
    "uk.gov.hmrc.mongo" %% "hmrc-mongo-test-play-28" % "0.50.0" % "test, it",
```

Reply:
Keep scalatest btw - that is still valid